src/views/start.njk

```diff
@@ -1,18 +1,117 @@
 {% extends "_base.njk" %}
 
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/accordion/macro.njk" import govukAccordion %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
 {% block content %}
   <h1 class="govuk-heading-xl">
-    Fit and Competent
+    Fit and Competent Controllers Registration
   </h1>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
+
       <p class="govuk-body">
-        Use this service to apply for a Fit and Competent licence
+        You must hold Fit and Competent Controller status if you wish to control deer:
+      </p>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li>
+          Out of season (under a specific <a class="govuk-link" rel="noreferrer noopener" target="_blank" href="http://gateway.snh.gov.uk/go/?id=35">5(6)</a> authorisation of the Deer Act)
+        </li>
+        <li>
+          At night (under the <a class="govuk-link" rel="noreferrer noopener" target="_blank" href="http://gateway.snh.gov.uk/go/?id=36">18(2)</a> authorisation of the Deer Act)
+        </li>
+        <li>
+          Under the <a class="govuk-link" rel="noreferrer noopener" target="_blank" href="http://gateway.snh.gov.uk/go/?id=34">general authorisation</a> (if you don’t meet the criteria below)
+        </li>
+      </ul>
+
+      {{ govukWarningText({
+        text: '' +
+          'You can use this service to apply, amend, renew or surrender your status.',
+        iconFallbackText: "Warning"
+      }) }}
+
+      <form method="post">
+        {{ govukButton({
+          text: "Continue",
+          classes: "govuk-!-margin-top-2 govuk-!-margin-bottom-8 naturescot-forward-button",
+          isStartButton: true
+        }) }}
+      </form>
+
+      {{ govukWarningText({
+        html: 'Any of the following who wish to control deer under the <a class="govuk-link" rel="noreferrer noopener" target="_blank" href="http://gateway.snh.gov.uk/go/?id=34">General Authorisation</a> ' +
+              '(to prevent damage to improved agricultural land or enclosed woodland) do not need to be registered:' +
```

Comment:
These lines are all...

```js
'lorem ipsum' +
'dolor sit amet' +
```

...but line 46 below is...

```js
'lorem ipsum
dolor sit amet'
```

...and I know it's picky, but I think we should use one style and stick to it. I don't have a recommendation, as I went looking at the GDS Style Guide and they deal with this sort of thing by using nunjucks blocks.

```html
{% set dontRegisterWarningHtml %}
Any of the foll...
...ed to be registered:
<ul>
    <li>...</li>
</ul>
{% endset -%}

{{ govukWarningText({
    html: dontRegisterWarningHtml
    iconFallbackText: "Warning"
}) }}
```

Feel free to do it any way you fancy, just be consistent.